oodocument/oodocument.py

```diff
@@ -70,10 +71,22 @@ def replace_with_index(self, data=[], dest=None, format=None, offset=0, word_nei
         else:
             self.convert_to(dest, format)
 
-    def replace_with_index_in_header(self, data=[], dest=None, format=None, offset=0):
+    def replace_with_index_in_header(
+        self,
+        data=[],
+        dest=None,
+        format=None,
+        offset=0,
+        word_neighbors=0,
+        style_name="Default Style",
+    ):
         data.sort(key=lambda x: x[0], reverse=True)
+
+        paragraph = self.document.getStyleFamilies().getByName("PageStyles").getByName(style_name).HeaderText
```

Comment:
We need to catch error here if it not exists.
Example error not finding header:
```
Traceback (most recent call last):
File "test.py", line 16, in <module>
oo.replace_with_index_in_header(data, "./output.pdf", "pdf", 0, neighbor_character, header_style_name)
File "/home/josx/Desarrollo/camba/proyectos/oodocument/oodocument/oodocument.py", line 84, in replace_with_index_in_header
paragraph = self.document.getStyleFamilies().getByName("PageStyles").getByName(style_name).HeaderText
oodocument.NoSuchElementException
```